py4DSTEM/io/nonnative/read_dm.py

```diff
@@ -28,39 +29,70 @@ def read_dm(fp, mem="RAM", binfactor=1, metadata=False, **kwargs):
                     metadata is read and returned, in the former case a DataCube
                     is returned with the metadata attached at datacube.metadata
     """
-    assert(isinstance(fp,(str,Path))), "Error: filepath fp must be a string or pathlib.Path"
-    assert(mem in ['RAM','MEMMAP']), 'Error: argument mem must be either "RAM" or "MEMMAP"'
-    assert(isinstance(binfactor,int)), "Error: argument binfactor must be an integer"
-    assert(binfactor>=1), "Error: binfactor must be >= 1"
+    assert isinstance(
+        fp, (str, Path)
+    ), "Error: filepath fp must be a string or pathlib.Path"
+    assert mem in [
+        "RAM",
+        "MEMMAP",
+    ], 'Error: argument mem must be either "RAM" or "MEMMAP"'
+    assert isinstance(binfactor, int), "Error: argument binfactor must be an integer"
+    assert binfactor >= 1, "Error: binfactor must be >= 1"
```

Comment:
Why this change?

Reply:
This is a pure formatting change from running [black](https://black.readthedocs.io/en/stable/the_black_code_style.html) on the file

Reply:
Hm, mkay.  Can you change this back please?  This looks silly and less readable to me.

Reply:
This is rather odd looking, I admit, so I've reverted it